shared/actions/typed-routes.tsx

```diff
@@ -3,12 +3,21 @@
 
 import * as RouteTreeGen from './route-tree-gen'
 
+// TODO i think this should go away. either we dispatch a team building action that's typed that produces these or
+// we actually type the routes / props
```

Comment:
> we dispatch a team building action that's typed that produces these
I think dispatching actions is too much for this. It's just a simple mapping. Dealing with actions has overhead.

> we actually type the routes / props
I'd prefer this. But until these can be properly typed, I think this is the best solution.